### DISCUSSION TITLE
如何去掉xmake输出内容的颜色代码?

### DISCUSSION THREAD

**Discussion by ZZH-Finalize** [xmake-io/xmake#5735] [Q&A]
xmake在执行中输出的文本信息带有颜色控制代码, 例如`\033[0m`这样的字符序列, 如果重定向xmake的输出到文件, 那么文件里就是一堆乱码, 有没有办法让xmake执行时不输出颜色代码只输出文本?

![image](https://github.com/user-attachments/assets/ea07ea54-329e-453e-83ac-919174ac51da)

**Comment by star-hengxing** [comment]
https://xmake.io/#/theme/builtin_themes?id=plain-theme
